### PR TITLE
[#7594][Platform] Assign universe UUID instead of backup UUID to target UUID while creating delete backup task.

### PR DESCRIPTION
**Pull request by mahendranbhat**
**Description:** 

When a backup is deleted, it's not listed on the Task history of that universe but when you go to Task (from left sidebar) which shows the Delete backup. The problem here is because the name of the "Delete backup" is not connected to the correct universe uuid. 

**Test plan:**

1. Created a universe
2. Taken backups
3. Deleted the backups 
4. The task history is showing deleted backups inside the universe and the universe name is correct in the task history page.

Sbt test

[info] Test run finished: 0 failed, 0 ignored, 10 total, 21.1s
[info] Passed: Total 10, Failed 0, Errors 0, Passed 10
[success] Total time: 46 s, completed 16 Jun, 2021 9:30:38 PM

### CODE REVIEW

**Review by jaydeepkumara — APPROVED** [review]

**Review by sb-yb — CHANGES_REQUESTED** [review]

**Review by sb-yb — APPROVED** [review]

File: managed/src/test/java/com/yugabyte/yw/controllers/BackupsControllerTest.java [hunk 1]
```diff
@@ -282,6 +283,8 @@ public void testDeleteBackup() {
     Result result = deleteBackup(resultNode, null);
     assertEquals(200, result.status());
     JsonNode json = Json.parse(contentAsString(result));
+    CustomerTask customerTask = CustomerTask.findByTaskUUID(fakeTaskUUID);
+    assertEquals(customerTask.getTargetUUID(), backup.getBackupInfo().universeUUID);
     assertEquals(json.get("taskUUID").size(), 1);
     assertAuditEntry(1, defaultCustomer.uuid);
   }
```
Comment (mahendranbhat) [inline managed/src/test/java/com/yugabyte/yw/controllers/BackupsControllerTest.java:290]: please add migration file to migrate old data.

**Review by mahendranbhat — COMMENTED** [review]
Please add migration file

File: managed/src/test/java/com/yugabyte/yw/controllers/BackupsControllerTest.java (hunk 1 above)
Comment (sb-yb) [reply managed/src/test/java/com/yugabyte/yw/controllers/BackupsControllerTest.java:290]: Was it in this state for some time already? Then yes. We should migrate.

**Comment by sb-yb** [comment]
I am curious if the unit tests pass after adding this migration script as the json seatures used look postgres specific. 
 If it gives problem then you can move this migration to postgres folder instead of common and add an empty migration in h2 subfolder.

**Comment by jaydeepkumara** [comment]
> I am curious if the unit tests pass after adding this migration script as the json seatures used look postgres specific.
> If it gives problem then you can move this migration to postgres folder instead of common and add an empty migration in h2 subfolder.

**Review by sb-yb — APPROVED** [review]

**Review by sb-yb — APPROVED** [review]

File: managed/ui/src/components/tables/ListBackups/ListBackups.js [hunk 2]
```diff
@@ -366,8 +366,8 @@ export default class ListBackups extends Component {
       .map((b) => {
         const backupInfo = b.backupInfo;
         if (
-          (backupInfo.actionType === 'CREATE' && backupInfo.status !== 'Deleted') ||
-          (showDeletedBackups && backupInfo.status === 'Deleted')
+          (backupInfo.actionType === 'CREATE' && b.state !== 'Deleted') ||
```
Comment (jaydeepkumara) [inline managed/ui/src/components/tables/ListBackups/ListBackups.js:369]: backupInfo.status for Deleted is unknown. So required to change to b.state(line number 273, we are assigning status value
).

**Review by mjoshi0923 — APPROVED** [review]

**Review by mjoshi0923 — APPROVED** [review]

**Comment by jaydeepkumara** [comment]
Summary:

When a backup is deleted, it's not listed on the Task history of that universe but when you go to Task (from left sidebar) which shows the Delete backup. The problem here is because the name of the "Delete backup" is not connected to the correct universe uuid.

Test plan:

Created a universe
Taken backups
Deleted the backups
The task history is showing deleted backups inside the universe and the universe name is correct in the task history page.

Sbt test
[info] Test run finished: 0 failed, 0 ignored, 10 total, 21.1s
[info] Passed: Total 10, Failed 0, Errors 0, Passed 10
[success] Total time: 46 s, completed 16 Jun, 2021 9:30:38 PM

Reviewers: jaydeepkumara, sb-yb, mjoshi0923, jitendra-12113

Reviewed By: jaydeepkumara, sb-yb, mjoshi0923

Differential Revision: https://github.com/yugabyte/yugabyte-db/pull/8529

**Pull request merged**